apps/omg_watcher_info/test/omg_watcher_info/db/block_test.exs

```diff
@@ -32,6 +32,31 @@ defmodule OMG.WatcherInfo.DB.BlockTest do
     end
   end
 
+  describe "get/1" do
+    @tag fixtures: [:initial_blocks]
+    test "retrieves a block by block hash" do
+      hash = "##{1000}"
+      block = hash |> DB.Block.get()
+
+      assert block.__struct__ == DB.Block
+      assert block.hash == hash
+    end
+
+    @tag fixtures: [:initial_blocks]
+    test "the transactions associated with a block can be preloaded" do
+      blknum = 1000
+
+      %{:transactions => transactions} =
+        DB.Repo.get(DB.Block, blknum)
```

Comment:
```suggestion
        block = DB.Repo.get(DB.Block, blknum)
        DB.Repo.preload(block,:transactions)
```

```suggestion
        DB.Repo.preload(DB.Repo.get(DB.Block, blknum),:transactions)
```

Reply:
Pre-loading transaction no longer-needed so this part of the code is removed. Thanks for code suggestions!